src/sentry/event_manager.py

```diff
@@ -566,6 +566,28 @@ def save(self, project_id, raw=False, assume_normalized=False):
         if transaction_name:
             transaction_name = force_text(transaction_name)
 
+        # Right now the event type is the signal to skip the group. This
+        # is going to change a lot.
+        if event.get_event_type() == 'transaction':
+            create_group = False
+        else:
+            create_group = True
+
+        org = Organization.objects.get_from_cache(
+            id=project.organization_id
+        )
```

Comment:
Isn't this already bound on [line 506](https://github.com/getsentry/sentry/blob/148a0c22f5f6fbea50f660cab78f93ebcec215a9/src/sentry/event_manager.py#L506-L507)?

Reply:
correct. will fix